channel/src/main/java/com/synopsys/integration/alert/channel/jira/common/JiraTestIssueRequestCreator.java

```diff
@@ -51,20 +52,29 @@
 
 public class JiraTestIssueRequestCreator implements TestIssueRequestCreator {
     private final Logger logger = LoggerFactory.getLogger(JiraTestIssueRequestCreator.class);
-    private final FieldUtility fieldUtility;
+
+    private final String customTopic;
+    private final String customMessage;
     private final JiraMessageParser jiraMessageParser;
 
     public JiraTestIssueRequestCreator(FieldUtility fieldUtility, JiraMessageParser jiraMessageParser) {
```

Comment:
Do we prefer a constructor for this over a static method?

Reply:
No. This is definitely an area that needs improvement, but changing this existing constructor would have only added to the already-excessive scope-creep.